src/pages/patientView/PatientViewPageUtils.tsx

```diff
@@ -51,3 +87,54 @@ export function getSamplesProfiledStatus(
         notProfiledIds,
     };
 }
+export function retrieveMutationalSignatureMap(
+    inputMetaData: GenericAssayMeta[]
+): MutationalSignatureLabelMap[] {
+    const mappedData = inputMetaData.map((metaData: GenericAssayMeta) => {
+        const nameSig: string = getGenericAssayMetaPropertyOrDefault(
+            metaData,
+            'MUTATION_TYPE',
+            ''
+        );
+        const classSig: string = getGenericAssayMetaPropertyOrDefault(
+            metaData,
+            'MUTATION_CLASS',
+            ''
+        );
+        const mutNameSig: string = getGenericAssayMetaPropertyOrDefault(
+            metaData,
+            'NAME',
+            ''
+        );
+        const signatureId = metaData.stableId;
+        return {
+            stableId: signatureId,
+            signatureLabel: nameSig,
+            signatureClass: classSig,
+            name: mutNameSig,
+        };
+    });
+    return mappedData;
+}
+
+export function createMutationalCountsObject(
```

Comment:
seems like this returns an array of objects so the function name should be pluralized